users/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json, jwt, requests
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+from django.conf import settings
+
+from .models import User
+
+class KakaotalkSignInView(View):
+    def get(self, request):
+        try : 
+            ACCESS_TOKEN        = request.headers.get('Authorization')
+            profile_information = requests.get(
+                "https://kapi.kakao.com/v2/user/me",
+                headers = {'Authorization' : f'Bearer {ACCESS_TOKEN}'}
+            )
+
+            profile_information_json = profile_information.json()
+            print(profile_information_json)
+            profile_id               = profile_information_json["id"]
+            email                    = profile_information_json["kakao_account"]["email"] 
+            nickname                 = profile_information_json["properties"]["nickname"]
+            profile_image            = profile_information_json["kakao_account"]["profile"]["profile_image_url"]
+
+            user, created     = User.objects.get_or_create(
+                social_id     = profile_id,
+                nickname      = nickname,
+                email         = email,
+                profile_image = profile_image)
+            siteToken         = jwt.encode({"id" : user.id}, settings.SECRET_KEY, algorithm=settings.ALGORITHM)
+            
+            if created: 
+                return JsonResponse({"Token" : siteToken}, status = 201)
+            else: 
+                return JsonResponse({"Token" : siteToken}, status = 200)
```

Comment:
프론트에서 계정이 새로 생겼을 때랑 그냥 로그인 했을 때랑 액션이 다른가요?? 다르다면 
```suggestion
            if created: 
                return JsonResponse({"Token" : siteToken}, status = 201)
            
            return JsonResponse({"Token" : siteToken}, status = 200)
```
다르지 않으면 통일 시켜주세요.
```suggestion
             
            return JsonResponse({"Token" : siteToken}, status = 200)
```

Reply:
액션 하나로 통일하였습니다.